pkg/driver/node.go

```diff
@@ -115,13 +113,7 @@ func (d *nodeService) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 	secrets := req.Secrets
 	mountOptions := []string{}
 	if opts, ok := volCtx["mountOptions"]; ok {
-		mountOptions = strings.Split(opts, ",")
-	}
-	for k, v := range options {
-		if v != "" {
-			k = fmt.Sprintf("%s=%s", k, v)
-		}
-		mountOptions = append(mountOptions, k)
+		mountOptions = append(mountOptions, opts)
```

Comment:
This is changing the logic. Has it been verified that mountOptions accepts.a comma separated string?

Before the change, it was accepting a string array.

Reply:
juicefs accepts `-o opt1=val1,opt2=val2,opt3=val3` because juicefs use `,` to separate string followed by `-o`, if `mountOptions` is an array like `["opt1=val1", "opt2=val2", "opt3=val3"]`, then juicefs will receive `-o opt1=val1 opt2=val2 opt3=val3`, the latter two are ignored. but I need to check `j.Mount` to see how `options` is passed first...

Reply:
```func makeMountArgs(source, target, fstype string, options []string) []string {
    // Build mount command as follows:
    //   mount [-t $fstype] [-o $options] [$source] $target
    mountArgs := []string{}
    if len(fstype) > 0 {
        mountArgs = append(mountArgs, "-t", fstype)
    }
    if len(options) > 0 {
        mountArgs = append(mountArgs, "-o", strings.Join(options, ","))
    }
    if len(source) > 0 {
        mountArgs = append(mountArgs, source)
    }
    mountArgs = append(mountArgs, target)

    return mountArgs
}
```

oops, it will join again, so there shouldn't be a bug here... I'll check more...